theano/gof/graph.py

```diff
@@ -223,27 +223,26 @@ def clone(self):
         return cp
 
     def clone_with_new_inputs(self, inputs, strict=True):
-        """
-        Duplicate this Apply instance in a new graph.
+        """Duplicate this `Apply` instance in a new graph.
 
         Parameters
         ----------
-        inputs
-            List of Variable instances to use as inputs.
-        strict : bool
-            If True, the type fields of all the inputs must be equal
-            to the current ones (or compatible, for instance Tensor /
-            GpuArray of the same dtype and broadcastable patterns,
-            in which case they will be converted into current Type), and
+        inputs: list of Variables
```

Comment:
In numpydoc, there should be a space to both sides of the colon: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring